src/openloco/graphics/gfx.h

```diff
@@ -145,19 +145,27 @@ namespace openloco::gfx
         uint8_t colour,
         string_id stringId,
         const void* args);
-    void draw_string_494B3F(
+    void draw_string_495224(
         drawpixelinfo_t& dpi,
         int16_t x,
         int16_t y,
+        int16_t width,
```

Comment:
Fix the original call; width is calculated at `0x00445EB1`.

Reply:
See #341.